js/form.js

```diff
@@ -62,7 +78,8 @@ const hasUniqueTags = (value) => {
 const hasValidCount = (value) => normalizeTags(value).length <= MAX_HASHTAG_COUNT;
 
 function onDocumentKeydown(evt) {
-  if (isEscapeKey && !isTextFieldFocused()) {
+  const isErrorMessageExists = Boolean(document.querySelector('.error'));
+  if (isEscapeKey && !isTextFieldFocused() && !isErrorMessageExists) {
```

Comment:
В других местах такая же ошибка
```suggestion
  if (isEscapeKey(evt) && !isTextFieldFocused() && !isErrorMessageExists) {
```